olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -190,13 +227,97 @@ class AuthenticationClient::Impl final {
       const AuthenticationCredentials& credentials,
       const std::string& userAccessToken, const SignOutUserCallback& callback);
 
+  client::CancellationToken IntrospectApp(std::string access_token,
+                                          IntrospectAppCallback callback);
+
   void SetNetworkProxySettings(
       const http::NetworkProxySettings& proxy_settings);
 
   void SetNetwork(std::shared_ptr<http::Network> network);
 
   void SetTaskScheduler(std::shared_ptr<TaskScheduler> task_scheduler);
 
+  static olp::authentication::IntrospectAppResult parseIntrospectAppResult(
```

Comment:
```suggestion
  static olp::authentication::IntrospectAppResult GetIntrospectAppResult(
```

Also, why is it a static method and not an anonymous function?